### ISSUE TITLE
Add How Do I See It block to visual works (to match linguistic works) (from 1963)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#71]
_Potential rescoping of issue:  it is object/works related, and becomes further complicated with addition of PMC photo archives.  We have other examples of non-Yale works (no object) records that should be considered along side this issue. - SP_ 
https://lux.collections.yale.edu/view/text/5ad5d97a-1b16-42c4-ad5e-145b2371643b 

------------------

**Problem Description:** 
Linguistic Objects show "How do I see It", with a link to the unit record, but Visual Works do not have this link.  

**Expected Behavior/Solution:**
Add a "How Do I See It" block to visual works that shows the same data as the corresponding object record.  At a minimum, this block should show the View this record on [unit] website link.

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Heather
- [ ] ~Committee discussions - Sarah~
- [ ] Feasibility/Team discussion - Sarah
- [ ] Check for other related tickets in forming (Where is it, how do I see it?) - Sarah
- [ ] Connect related tickets - Sarah
- [x] Backend requirements - NONE
- [ ] Frontend requirements- TBD

**Questions:**
TBD

**Requirements:**
TBD: List of details required for the completion of the issue or requirements for the feature/bug. This can also include requirements that lie outside of the teams such as new design docs or clarification from an outside source.

**UAT/LUX Examples:**

- Visual Work: https://lux-front-tst.collections.yale.edu/view/visual/d743bb92-4ae7-492a-8715-a0a67ed75507
- Corresponding Visual Object: https://lux-front-tst.collections.yale.edu/view/object/3e8af430-665d-47a2-b043-7170e2d7ff1c
- Linguistic Work: https://lux-front-tst.collections.yale.edu/view/text/5740c505-7ac6-4cf6-a502-2cc8b2de6c37

**Dependencies/Blocks: NONE**

**Related Github Issues: Where is it at Yale tickets.**

- [x] https://git.yale.edu/lux-its/lux-web/issues/944
- [x] https://git.yale.edu/lux-its/lux-web/issues/1359

**Related links:**

- [(https://www.bugherd.com/projects/284041/tasks/1865)](https://www.bugherd.com/projects/284041/tasks/1865)

**Wireframe/Mockup:**
![image](https://github.com/project-lux/lux-frontend/assets/160149818/adf20ff5-2724-4d17-88b1-9f8df4bdbcc4)

**Comment by roamye** [comment]
I went to the other conversation and do not see any comments on why this is blocked. Additionally, it says "Blocks: None". So why is this blocked? Is it because of the note at the top? Should this be unblocked? 

@prowns @kamerynB

**Comment by kamerynB** [comment]
@roamye It may have been blocked at the time but going back to the original issue in the private repo, the 2 blocking issues have been completed and closed. I will move this out of the Blocked column and into the On Deck column.

**Comment by prowns** [comment]
Validating scope of ticket at UAT.

**Comment by prowns** [comment]
Propose close.   YUL links to bibliographic records.  Visual items do not have links to works - won't fix.
